backend/src/main/java/com/bakdata/conquery/models/config/XodusStoreFactory.java

```diff
@@ -213,19 +205,10 @@ private void configureMapper(ConqueryConfig config) {
 		return storages;
 	}
 
-	private List<String> getRelativePathElements(Path path) {
-		ArrayList<String> list = new ArrayList<>();
-		Path relative = getDirectory().relativize(path);
-		for (int i = 0; i < relative.getNameCount(); i++) {
-			list.add(relative.getName(i).toString());
-		}
-		return list;
-	}
-
 	private boolean environmentHasStores(File pathName) {
 		Environment env = findEnvironment(pathName);
-		boolean exists = env.computeInTransaction(t -> env.storeExists(StoreInfo.DATASET.getName(), t));
-		env.computeInTransaction(t -> env.getAllStoreNames(t));
+		boolean exists = env.computeInTransaction(t -> env.storeExists(StoreInfo.DATASET.storeInfo().getName(), t));
```

Comment:
jetzt wo ich sehe, wie das StoreInfo verwendet wird ist das iwie auch doppelt gemoppelt?

Reply:
bzw, ist die Zeile richtig überhaupt?

Reply:
Eigentlich müsste hier nach allen Stores getestet werden anstatt nur nach einem

Reply:
Du kannst in einem Environment die Stores auflisten lassen

Reply:
Ich checke jetzt alle stores für namespaced